ui/src/org.test.ts

```diff
@@ -0,0 +1,91 @@
+import { getOrg, orgIdValidationStore } from "./org"
+import { ValidationStatus } from './validation'
+import { get } from 'svelte/store'
+
+jest.mock("./api")
+
+describe("fetching an org", () => {
+  it("returns an org", async () => {
+
+    const promise = await getOrg("radicle")
+    expect(promise).toEqual({
+      id: "radicle",
+      shareableEntityIdentifier: "radicle@123abcd.git",
+      avatarFallback: {
+        background: {
+          r: 255, g: 67, b: 34
+        },
+        emoji: "🔥"
+      }
+    })
+  })
+})
+
+describe("validation", () => {
```

Comment:
This is nice, we'll be able to remove the bulk of the integration validation tests and make the integration test suite faster.